examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -2180,14 +2180,14 @@ server cluster PowerSource = 47 {
   attribute(readonly) int8u batteryPercentRemaining = 12;
   attribute(readonly) int32u batteryTimeRemaining = 13;
   attribute(readonly) enum8 batteryChargeLevel = 14;
-  attribute(readonly) ENUM8 activeBatteryFaults[] = 18;
+  attribute(readonly) ENUM8 activeBatteryFaults[8] = 18;
```

Comment:
There is a concrete enum for man of these "enum8". The types descend from enum8, but are concrete enums

Reply:
> There is a concrete enum for man of these "enum8". The types descend from enum8, but are concrete enums

In the spec, yes.  The XML we have says:
```
<attribute side="server" code="0x0012" define="POWER_SOURCE_ACTIVE_BAT_FAULTS" type="ARRAY" entryType="ENUM8" length="8" writable="false" optional="true">ActiveBatteryFaults</attribute>
```
and the IDL is faithfully reflecting that....

Reply:
Oh yes ... here is where having an IDL starts paying dividents.  Noticing this kind of stuff.